app/users/resources.py

```diff
@@ -0,0 +1,95 @@
+import json
+
+from flask import request
+from flask_restful import Resource
+
+from app import db, User
+from app.lib.response_util import buildOkResponse
+
+
+def returnUser(user):
+    """Private method to convert a user model into a dictionary.
+
+        Args:
+        user: A user model.
+    """
+
+    return {
+        "id": user.id, "name": user.name
```

Comment:
It is easier to read dictionaries when their key-value pairs are separated by new lines.
```
return {
    "id": user.id,
    "name": user.name
}
```